src/app/lib/layoutGrid/index.js

```diff
@@ -195,6 +203,8 @@ export const nestedGridItemSmallCss = css`
     grid-column: ${props => props.gridColumnStart} / span
       ${props =>
         specifiedOrMaximum(props.gridSpan.group3 || props.gridSpan.default, 5)};
+    margin-left: ${props =>
+      (props.marginLeft && props.marginLeft.group3) || ''};
```

Comment:
@RayNjeri  Thanks for addressing the extra spacing issue by pairing with UX!

We should look at a more scalable solution than overriding the margin left on very specific breakpoints for the Medium & Small grid items layout. However, we will fix forwards, in this issue: https://github.com/bbc/simorgh/issues/2149, since the work here will be a great improvement on our current images & will be needed for the Video work, too.